pkg/cmd/project/item-edit/item_edit.go

```diff
@@ -28,6 +28,7 @@ type editItemOpts struct {
 	date                 string
 	singleSelectOptionID string
 	iterationID          string
+	clear                bool
```

Comment:
@arunsathiya : what do you think about adding to `Example` showing and commenting about the `--clear` flag?

Reply:
That definitely is a good idea! I have updated in https://github.com/cli/cli/pull/8024/commits/9d81ca0e44f9172b43ae41d2deaaa092bc7c3fcc.

I have also added a note in the longform documentation that clear flag can be used, is that alright? - https://github.com/cli/cli/pull/8024/commits/eebef99b1cd4e6fc28896c500eda7cfc397b55ff